ynput/OpenPype
# nuke: split write node product instance features

## Changelog Description
Improving Write node product instances by allowing precise activation of specific features.

## Additional info
- abstracting the way linked knobs are prepared from settings

## Testing notes:
1. open settings at `project_settings/nuke/create/CreateWriteRender/instance_attributes`
2. notice that more options were added 
![image](https://github.com/ynput/OpenPype/assets/40640033/8f43f9d2-96f7-4f83-9b7c-3b42354f3b5f)
3. set any of new features and save settings
4. open nuke within the project you had just configured
5. notice that set features are showing within the linked write node features
in my case it looks like this 
![image](https://github.com/ynput/OpenPype/assets/40640033/f62fde2a-0a1a-4599-9a2d-815ef8f2b3c7)
